sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,252 @@
+
+**Protocol-Mandated Tasks:**
+- Signers must accept and process BTC deposit requests.
+- They must fulfill BTC withdrawal requests in a timely manner, ensuring accurate execution.
+- Signers are responsible for moving BTC to a new UTXO when private keys are rotated.
+- Signers must perform UTXO consolidation as BTC is deposited to optimize the number of unspent outputs [1].
+- Signers are required to deduct transaction fees from users to fund BTC withdrawal transactions. This includes:
+  - Ensuring that the transaction fee is deducted from the user.
+  - Setting a minimum sBTC withdrawal amount to cover the estimated transaction fees.
+  - Estimating the transaction fee proportionally based on the requested operation [2].
+ 
+**Operational Best Practices:**
+- Signers must maintain industry-standard operational security (opsec) around hosts and private data, including private keys.
+- They should collectively coordinate to calculate and advertise the fee parameters of the system, including:
+  - The minimum sBTC peg-out amount.
+  - The STX transaction fee for minting sBTC. This fee is paid by the user and can be sponsored by a 3rd party.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- Does the proposed sBTC signer have a demonstrable operating history which shows their experience and reliability in running blockchain services?
+- Has the proposed sBTC signer participated in running a Stacks signer instance on Stacks 2.5 testnet or mainnet, and can they provide metrics showing this (ex: amount of stacks stacked over past several cycles)?
+  * Note: The sBTC signer is a Stacks event observer, meaning that the experience of running a Stacks node signer directly translates to running an sBTC signer.
+- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer? 
+  * Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.
+- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
+- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.
```

Comment:
I'm sure this has already been considered, but it might be worth explicitly stating 'geographical distribution' as a criterion for the overall set of signers criteria/consideration. Without this, it's possible that each of the 15 signers could meet individual eligibility criteria, but the entire group could still be U.S.-based, which could pose a jurisdictional risk.

Reply:
Yes, thanks Hiro. This should have been included (it was one of the factors we've considered). I'll update the SIP.

Reply:
resolved 02df3fde8c1d84c0a280a120edda322285503ee4